### PR TITLE
make the log path configurable

### PR DESCRIPTION
**Pull request by btang-aa**
Let's read the path from Cypress.json env, if not exist, by default, use `cypress/logs`

### CODE REVIEW

**Comment by drewbrend** [comment]
I would like to see this merged.  I don't think you should be changing the formatting (quotes, spacing, etc), it now differs from the rest of the files in this repo and makes it harder to review your change when formatting changes are also shown together.

**Comment by ffesseler** [comment]
Yes, I'm interested to see this merged too.
BTW, I see the log folder is retrieved from cypress env. variables. Would it not be better to declare it as a config option like the screenshotsFolder or integrationFolder ?

**Comment by ffesseler** [comment]
I'm keen on doing a MR but still waiting for an answer about what is the recommended way to do it.

File: src/index.js [hunk 1]
```diff
@@ -5,7 +5,10 @@ const path = require('path')
 const debug = require('debug')('cypress-failed-log')
 
 const cleanupFilename = s => Cypress._.kebabCase(Cypress._.deburr(s))
-const getFilepath = filename => path.join('cypress', 'logs', filename)
+const getFilepath = filename =>
+  Cypress.env('logFolder')
+    ? path.join(Cypress.env('logFolder'))
```
Comment (bahmutov) [inline src/index.js]: did you forget to join logFolder and filename here?

**Comment by bahmutov** [comment]
Closing as abandoned

**Pull request closed without merging**